src/styles/header.scss

```diff
@@ -0,0 +1,62 @@
+body {
+  font-family: "Roboto", sans-serif;
+}
+
```

Comment:
I see you have same body selector in `reset.scss`, so move `font-family` there and delete body selector in `header.scss`